src/_layouts/home.liquid

```diff
@@ -22,8 +22,12 @@
     <div class="app-hero">
       <div class="container">
         <small class="app-hero__subheading">Bangkok, Thailand</small>
-        <div class="app-hero__heading display-1">9-10 December, 2022</div>
-        <div class="app-hero__heading display-4">Thank you for joining us!</div>
+        <div class="app-hero__heading display-1">October 6-7, 2023</div>
```

Comment:
maybe just write October, 2023 until we lock in the dates?

Reply:
The sponsor deck also has these dates and we will need to lock these dates for the venue by mid-next month so it would be fine to use these dates.
![image](https://user-images.githubusercontent.com/696529/213646174-d3f5851e-41da-409d-b810-181beac550f5.png)